### PR TITLE
Update 2021-10-21-hayat21a.md

### PR DESCRIPTION
**Pull request by nasir6**

### CODE REVIEW

File: _posts/2021-10-21-hayat21a.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ publisher: PMLR
 issn: 2640-3498
 id: hayat21a
 month: 0
-tex_title: Multi-Label Generalized Zero Shot Learning for the Classiffcation of Disease
+tex_title: Multi-Label Generalized Zero Shot Learning for the Classiffication of Disease
```
Comment (lawrennd) [inline _posts/2021-10-21-hayat21a.md:33]:
```suggestion
tex_title: Multi-Label Generalized Zero Shot Learning for the Classification of Disease
```

**Review by lawrennd — COMMENTED** [review]
Did you mean to change spelling to one f as shown?

The correction(s) also needs to be applied to the "title" field.

File: _posts/2021-10-21-hayat21a.md (hunk 1 above)
Comment (nasir6) [reply _posts/2021-10-21-hayat21a.md:33]: I opened a new pull request with correct spelling of "Classification"

**Comment by nasir6** [comment]
#3

**Pull request closed without merging**